pkg/kubectl/cmd/rollout/rollout_history.go

```diff
@@ -49,6 +49,9 @@ var (
 func NewCmdRolloutHistory(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	options := &HistoryOptions{}
 
+	validArgs := []string{"deployment"}
```

Comment:
@fabianofranz You should think of a way for us to refactor this to have these completions work well for us.  I don't think it needs to be resolved in this pull, but we'll want some way to add `dc` and `deploymentconfig` here.